packages/@o3r/rules-engine/src/stores/rulesets/rulesets.selectors.spec.ts

```diff
@@ -138,8 +163,9 @@ describe('RuleSets Selector tests', () => {
       requestIds: []
     };
     const allRuleSetsArray = selectors.selectAllRulesets.projector(state);
+    const rulesetsInRange = selectors.selectRuleSetsInRange.projector(allRuleSetsArray);
 
-    expect(selectors.selectActiveRuleSets.projector(allRuleSetsArray)).toEqual(['r3']);
+    expect(selectors.selectActiveRuleSets.projector(rulesetsInRange)).toEqual(['r3']);
```

Comment:
Does this change mandatory? In the end  `allRuleSetsArray` is reached behind `selectRuleSetsInRange`

Reply:
it is mandatory because the input selector of `selectActiveRuleSets`. It is no longer `selectAllRulesets` but `selectRuleSetsInRange`